app/graphql/types/sample_type.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Types
+  # Sample Type
+  class SampleType < Types::BaseObject
+    implements GraphQL::Types::Relay::Node
+    description 'A sample'
+
+    field :description, String, null: true, description: 'Description of the sample.'
+    field :name, String, null: false, description: 'Name of the sample.'
+    field :project_id, String, null: false, description: 'Project the sample is on.'
```

Comment:
**nitpick (blocking):** Can you remove `project_id` field, we are trying to stay away from showing and using these. Instead can you make this a `project` field like how project has a parent field.

Reply:
Fixed.